tilelang/tools/lower_trace/core.py

```
@@ -0,0 +1,1024 @@
+        import tvm.ffi
+
+        for ffi_name in _CODEGEN_FFI_NAMES:
+            try:
+                orig = tvm.ffi.get_global_func(ffi_name)
+                if orig is not None:
+                    wrapped = _wrap_codegen_ffi(orig)
+                    wrapped._original_ffi_name = ffi_name
+                    _original_codegen_ffis[ffi_name] = orig
+                    tvm.ffi.register_global_func(ffi_name, wrapped, override=True)
+            except Exception:
+                pass
```

Ruff (0.15.18)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)